cmd/fs-v1-metadata.go

```diff
@@ -41,37 +41,38 @@ const (
 	fsMetaVersion100 = "1.0.0"
 
 	// FS backend meta 1.0.1 version.
-	fsMetaVersion = "1.0.1"
+	fsMetaVersion101 = "1.0.1"
 
-	// FS backend meta format.
-	fsMetaFormat = "fs"
+	// FS backend meta 1.0.2 version.
```

Comment:
We can add a comment saying which fields were removed and which were added into fsMetaV1 @poornas

Reply:
Removed the fields "Format" and "Minio" from fsMetaV1 as they were unused. Added "Checksum" field - to be used in future for bit-rot protection.

Reply:
ok